README.md

```diff
@@ -7,31 +7,95 @@
+
+- 🧩 [@code-pushup/eslint-plugin](./packages/plugin-eslint#readme)
+  - Static analysis using **ESLint** rules
+- 🧩 [@code-pushup/coverage-plugin](./packages/plugin-coverage#readme)
+  - **Test Coverage** analysis
+- 🧩 [@code-pushup/js-packages-plugin](./packages/plugin-js-packages#readme)
+  - Package audit on **Security** and **Outdates**
+- 🧩 [@code-pushup/lighthouse-plugin](./packages/plugin-lighthouse#readme)
+  - web performance and best practices from **Lighthouse**
+
+---
+
+<h3>Custom Plugins</h3>
+
+- 🛠️ [Integrate your own metric with custom plugins](./packages/cli/README.md#getting-started)
+  - Example plugins repo
+  - Docs
+- 📦 [@code-pushup/models](./packages/models#readme)
+  - **schemas and types** for data models (useful for custom plugins or other integrations
+- 📦 [@code-pushup/utils](./packages/utils#readme)
+  - various **utilities** (useful for custom plugins or other integrations)
+  <!-- [Get enterprise support](https://code-pushup.dev/enterprise-support) -->
+
+<h2>How It Works!</h2>
+
+<ol>
+<li>
+<b>Configure</b><br/>
+Pick form a set of supported packages or include your own ideas.
+</li>
+<li>
+<b>Integrate</b><br/>
+Use our integration guide and packages to setup CI integration in minutes.
+</li>
+<li>
+<b>Observe</b><br/>
+Guard regressions and track improvements with every code change.
+</li>
+<li>
+<b>Wing it!</b><br/>
+Watch improve, share reports
+</li>
+</ol>
+
+<table cellpadding="0" cellspacing="0" border="0">
+<tr>
+<td>
+<img alt="Code-Pushup Logo" src="./packages/cli/docs/images/code-pushup-logo.png"> 
+</td>
+<td>
+<h2>Just follow the score</h2>
+<h3>See regressions - Report scores - On every PR</h3>
+<!-- link when web landing is ready <a href="./packages/cli/README.md#getting-started">Try our paid features</a> -->
+
+</td>
+</tr>
+
+</table>
 
-- [📦 @code-pushup/cli](./packages/cli#readme) - **CLI** for **collecting** audit results and **uploading** report to portal
-- [📦 @code-pushup/core](./packages/core#readme) - implementation of **core business logic** (useful for custom integrations)
-- [📦 @code-pushup/models](./packages/models#readme) - **schemas and types** for data models (useful for custom plugins or other integrations)
-- [📦 @code-pushup/utils](./packages/utils#readme) - various **utilities** (useful for custom plugins or other integrations)
-- plugins:
-  - [🧩 @code-pushup/eslint-plugin](./packages/plugin-eslint#readme) - static analysis using **ESLint** rules
-  - [🧩 @code-pushup/coverage-plugin](./packages/plugin-coverage#readme) - code coverage analysis
-  - [🧩 @code-pushup/js-packages-plugin](./packages/plugin-js-packages#readme) - package audit and outdated dependencies
-  - [🧩 @code-pushup/lighthouse-plugin](./packages/plugin-lighthouse#readme) - web performance and best practices from **Lighthouse**
+<h2>Want to support us?</h2>
 
-If you want to contribute, please refer to [CONTRIBUTING.md](./CONTRIBUTING.md).
+<ul>
+<li><a href="./CONTRIBUTING.md">Contribute</a></li>
```

Comment:
When I first read this, I thought `Contribute` was asking for donations :sweat_smile: